### PR TITLE
fix: left panel is reopened after closing panel with pulling completed

### PR DESCRIPTION
**Pull request by crui3er**

### CODE REVIEW

File: src/dotnet/App.Server/appsettings.Development.json [hunk 1]
```diff
@@ -65,9 +65,9 @@
       // "Microsoft.EntityFrameworkCore.Database": "Warning",
       // "Microsoft.EntityFrameworkCore.Database.Command": "Debug",
       // "Microsoft.Hosting.Lifetime": "Warning",
-      "Stl": "Warning",
-      "Stl.Fusion": "Information",
-      "Stl.Rpc": "Debug"
+      "ActualLab": "Warning",
```
Comment (alexyakunin) [inline src/dotnet/App.Server/appsettings.Development.json:68]: Nice :)

Comment (alexyakunin) [reply src/dotnet/App.Server/appsettings.Development.json:68]: I'll take a look if something else like this is there.

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanel.razor [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <SideNav
     Side="SideNavSide.Left"
     IsOpen="@_isOpen"
-    VisibilityChanged="@(isOpen => PanelsUI.Left.SetIsVisible(isOpen))">
+    VisibilityChanged="@(NoStateHasChanged.EventCallback<bool>(isOpen => PanelsUI.Left.SetIsVisible(isOpen)))">
```
Comment (alexyakunin) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanel.razor:8]: Is this because of some weird side effect of async-like change?

**Review by alexyakunin — APPROVED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanel.razor (hunk 2 above)
Comment (crui3er) [reply src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanel.razor:8]: It's just to get rid of redundant re-renders.

**Pull request merged**